tests/e2e/config/bootstrap.js

```diff
@@ -296,7 +316,7 @@ beforeAll( async () => {
 		if ( res.status() > 399 ) {
 			const req = res.request();
 			// eslint-disable-next-line no-console
-			console.warn( res.status(), req.method(), req.url() );
+			console.debug( res.status(), req.method(), req.url() );
```

Comment:
Note, the `pageerror` above is kept on `console.error` since these should require a manual exception as they generally are not expected.

https://github.com/google/site-kit-wp/blob/c5e2acfef02e82ecf5752e52368bf90949249e57/tests/e2e/config/bootstrap.js#L302-L304